mycroft/skills/intent_service.py

```diff
@@ -37,6 +37,89 @@ def __init__(self, emitter):
         self.emitter.on('recognizer_loop:utterance', self.handle_utterance)
         self.emitter.on('detach_intent', self.handle_detach_intent)
         self.emitter.on('detach_skill', self.handle_detach_skill)
+        self.emitter.on('converse_status_response', self.handle_conversation_response)
+        self.emitter.on('intent_request', self.handle_intent_request)
+        self.emitter.on('intent_to_skill_request', self.handle_intent_to_skill_request)
+        self.emitter.on('active_skill_request', self.handle_active_skill_request)
```

Comment:
We should try to group these into `mycroft.intent.[name]` to keep them grouped. (The older ones should be renamed as well but that's not required since that's outside the scope of this PR)